version.gradle

```diff
@@ -1,14 +1,14 @@
 ext {
```

Comment:
While reviewing the other PR https://devtopia.esri.com/runtime/kotlin/pull/3392/files, I was verifying myself to see if samples, toolkit and kotlin repo versions.gradle match. came across this PR
I noticed, below three version that need to be updated to match the other version upgrades?
   ```
 androidBrowserVersion = '1.5.0'
    // Plugin versions
    gradleVersion = '8.1.1'
    // Java version
    javaVersion = 1.17
```
to 
   ```
 androidBrowserVersion = '1.7.0'
    // Plugin versions
    gradleVersion = '8.2.0'
    // Java version
    javaVersion = 1.8
```